landingzones/views_ajax.py

```diff
@@ -22,7 +22,8 @@ def check_zone_permission(self, zone, user):
         return user.has_perm(permission, obj=zone.project)
 
     def post(self, request, *args, **kwargs):
-        zone_uuids = request.data.getlist('zone_uuids[]')
+        zone_uuids_string = request.data.get('zone_uuids')
+        zone_uuids = zone_uuids_string.split(',')[:-1]
```

Comment:
As commented in the javascript file, we should still provide and grab a standard list like in the original implementation. Here in the view we just need to check if the value we're looking at is an instance of `QueryDict` before we call `getlist()`.